mkdocs.yml

```
@@ -38,7 +38,7 @@ markdown_extensions:
   - admonition
   - attr_list
   - pymdownx.tabbed:
-      alternate_style: true
+      alternate_style: true 
```

```suggestion
      alternate_style: true
```

I guess this trailing whitespace is spurious?

---

Correct. I had to do a merge because Ruben did the same fix as me for the mkdoc.yml file and I thought "oh I'll just do that in the browser". Good catch.